.github/workflows/release-step-1.yml

```diff
@@ -250,21 +251,23 @@ jobs:
             NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
         steps:
-            - name: Checkout public master branch
+            - name: Checkout master branch
               uses: actions/checkout@v3
               with:
                   fetch-depth: 0
+                  repository: ${{ github.repository }}
+                  token: ${{ secrets.MP_SEMANTIC_RELEASE_BOT }}
```

Comment:
These aren't needed currently, is there a reason they are needed moving forward?